tests/e2e/ambient/ambient_suite_test.go

```diff
@@ -64,4 +71,29 @@ func setup() {
 	Expect(err).NotTo(HaveOccurred())
 
 	k = kubectl.New()
+	clr = cleaner.New(cl)
 }
+
+var _ = BeforeSuite(func(ctx SpecContext) {
+	clr.Record(ctx)
+	Expect(k.CreateNamespace(operatorNamespace)).To(Succeed(), "Namespace failed to be created")
+
+	if skipDeploy {
+		Success("Skipping operator installation because it was deployed externally")
+	} else {
+		Expect(common.InstallOperatorViaHelm()).
+			To(Succeed(), "Operator failed to be deployed")
+	}
+
+	Eventually(common.GetObject).WithArguments(ctx, cl, kube.Key(deploymentName, operatorNamespace), &appsv1.Deployment{}).
+		Should(HaveConditionStatus(appsv1.DeploymentAvailable, metav1.ConditionTrue), "Error getting Istio CRD")
```

Comment:
```suggestion
		Should(HaveConditionStatus(appsv1.DeploymentAvailable, metav1.ConditionTrue), "Operator not deployed")
```

I know it's not quite part of your change, but since you're already touching this code shouldn't it have the correct error message in case this fails?